docker/Dockerfile

```diff
@@ -3,31 +3,50 @@ FROM ros:melodic-perception
 ENV ROSUSER="user"
 
 RUN adduser --home /home/$ROSUSER --shell /bin/bash --gecos '' --disabled-password $ROSUSER \
-    && usermod -a -G root,sudo $ROSUSER \
+    && usermod -a -G root,sudo,video, tty, dialout $ROSUSER \
```

Comment:
Copying files from `/etc/skel' ...
usermod: group '' does not exist
The command '/bin/sh -c adduser --home /home/$ROSUSER --shell /bin/bash --gecos '' --disabled-password $ROSUSER     && usermod -a -G root,sudo,video, tty, dialout $ROSUSER     && echo "$ROSUSER ALL=(ALL) NOPASSWD:ALL" > /etc/sudoers.d/$ROSUSER' returned a non-zero code: 6